app/views/customers/_customer.html.erb

```diff
@@ -15,3 +15,14 @@
   </p>
 
 </div>
+
+<% if @customer.present? %>
```

Comment:
The code above is expecting a `customer` local variable, but your new code is expecting a `@customer` instance variable. Something is wrong here, and I don't think you even need to change this file.